content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,155 @@ aliases:
-supports them. Following sections describe each of the traffic management API resources in
-more detail.
-
-### Traffic routing use cases
-
-You might use all or only some of the Istio traffic management API resources,
-depending on your use case. Istio handles basic traffic routing by default,
-but configurations for advanced use cases might require the full range of Istio
-traffic routing features.
-
-#### Routing traffic to multiple versions of a service {#routing-versions}
-
-Typically, requests sent to services use a service's hostname or IP address,
-and clients sending requests don't distinguish between different versions of
-the service.
-
-With Istio, because the Envoy proxy intercepts and forwards all requests and
-responses between the clients and the services, you can use routing rules with
-[service subsets](/docs/concepts/traffic-management/#service-subsets)
-in a virtual service to configure the [routing rules](/docs/concepts/traffic-management/#routing-rules)
-for multiple versions of a service.
-
-Service subsets are used to label all instances that correspond to a specific
-version of a service.
-Before you configure routing rules, the Envoy proxies use round-robin load
-balancing across all service instances, regardless of their subset. After you
-configure routing rules for traffic to reach specific subsets, the Envoy
-proxies route traffic to the subset according to the rule but again use
-round-robin to route traffic across the instances of each subset.
-
-This configuration method provides the following advantages:
-
-- Decouples the application code from the evolution of the application's dependent services.
-- Provides monitoring benefits.
-For details, see [Mixer policies and telemetry](/docs/reference/config/policy-and-telemetry/).
-
-For example, in A/B testing we often want to configure traffic routes based on
-percentages. With Istio, you can use a virtual service to specify a routing
-rule that sends 25% of requests to instances in the `v2` subset, and sends the
-remaining 75% of requests to instances in the `v1` subset. The following
-configuration accomplishes our example for the `reviews` service.
+-   [Introducing Istio Traffic Management](/docs/concepts/traffic-management/#introducing-istio-traffic-management):
+    Learn about how Istio helps you control the flow of traffic in your mesh
+    with simple, flexible configuration.
+
+-   [Network resilience and testing](/docs/concepts/traffic-management/#network-resilience-and-testing):
+    Learn about Istio's dynamic failure recovery features that you can configure
+    to test and build tolerance for failing nodes, and to prevent cascading
+    failures to other nodes.
+
+-   [Architecture](/docs/concepts/traffic-management/#architecture): Learn about
+    Pilot, Istio's core traffic management component and Envoy proxies and how
+    they enable service discovery and traffic control for services in the mesh.
```

Comment:
I think this should be second

Reply:
Strongly disagree. In my original proposal I actually suggesting moving this content out of the doc entirely (into a separate architecture section for potential contributors & interested parties); though we agreed in the docs group meeting that we'd keep it at the end of this doc for now, largely because we haven't reorganized similar content out of any of the other concepts docs. In the current structure we're entirely focused on features and how they help the user until the arch. section at the end.

Reply:
I think the 3 bullets at the top of the page obscure the fact that the document is feature focused, with architecture stuff moved to the back. Maybe the bullets aren't needed at all? Otherwise, I think it's fine for now, until we have a proper architecture section of the docs that we move the architecture section to.

Reply:
That's a good point too: I just reused the original bullets (with a bit of tweaking), I'm not entirely sure how much they help orient the reader beyond the TOC (which they can see on the right) and the actual intro.

Reply:
I suggest removing them.

Reply:
Taken them out!